src/styles/h5p-components.css

```diff
@@ -115,7 +115,7 @@
 /* CARDS */
 .h5p-theme .h5p-cardholder {
   background-color: var(--h5p-theme-ui-base);
-  border: 0.1em solid transparent;
+  border: 0.2em solid transparent;
```

Comment:
@Gwenillia `0.2em` will in many cases result in `3px`, but is there a reason why you deviated from the ticket specification that asked for `3px`? And you say `3px` in your commit message. :thinking: If someone uses a larger font size, say `18px`, the border would become `4px`.